docs/admin/notifications.md

```diff
@@ -0,0 +1,251 @@
+for more options.
+
+## Webhook
+
+The webhook delivery method sends an HTTP POST request to the defined endpoint. The purpose of webhook notifications is
+to enable integrations with other systems.
+
+**Settings**:
+
+| Required | CLI                                | Env                                    | Type  | Description                             |
+|:--------:|------------------------------------|----------------------------------------|-------|-----------------------------------------|
+|    ✔️    | `--notifications-webhook-endpoint` | `CODER_NOTIFICATIONS_WEBHOOK_ENDPOINT` | `url` | The endpoint to which to send webhooks. |
+
+Here is an example payload for Coder's webhook notification:
+
+```json
+{
+    "_version": "1.0",
+    "msg_id": "88750cad-77d4-4663-8bc0-f46855f5019b",
+    "payload": {
+        "_version": "1.0",
+        "notification_name": "Workspace Deleted",
+        "user_id": "4ac34fcb-8155-44d5-8301-e3cd46e88b35",
+        "user_email": "danny@coder.com",
+        "user_name": "danny",
+        "user_username": "danny",
+        "actions": [
+            {
+                "label": "View workspaces",
+                "url": "https://et23ntkhpueak.pit-1.try.coder.app/workspaces"
+            },
+            {
+                "label": "View templates",
+                "url": "https://et23ntkhpueak.pit-1.try.coder.app/templates"
+            }
+        ],
+        "labels": {
+            "initiator": "danny",
+            "name": "my-workspace",
+            "reason": "initiated by user"
+        }
+    },
+    "title": "Workspace \"my-workspace\" deleted",
+    "body": "Hi danny\n\nYour workspace my-workspace was deleted.\nThe specified reason was \"initiated by user (danny)\"."
+}
+```
+
+The top-level object has these keys:
+
+- `_version`: describes the version of this schema; follows semantic versioning
+- `msg_id`: the UUID of the notification (matches the ID in the `notification_messages` table)
+- `payload`: contains the specific details of the notification; described below
+- `title`: the title of the notification message (equivalent to a subject in SMTP delivery)
+- `body`: the body of the notification message (equivalent to the message body in SMTP delivery)
+
+The `payload` object has these keys:
+
+- `_version`: describes the version of this inner schema; follows semantic versioning
+- `notification_name`: name of the event which triggered the notification
+- `user_id`: Coder internal user identifier of the target user (UUID)
+- `user_email`: email address of the target user
+- `user_name`: name of the target user
+- `user_username`: username of the target user
+- `actions`: a list of CTAs (Call-To-Action); these are mainly relevant for SMTP delivery in which they're shown as
+  buttons
+- `labels`: dynamic map of zero or more string key-value pairs; these vary from event to event
+
+## Preferences (enterprise)
```

Comment:
Let's call this "Delivery Preferences"

I'll add a section on turning notifications on/off at the user level.